src/executables/csql.c

```diff
@@ -1460,6 +1464,205 @@ csql_do_session_cmd (char *line_read, CSQL_ARGUMENT * csql_arg)
 	}
 
       break;
+
+    case S_CMD_CONNECT:
+      /*dbname_cpy can be stored DB name+ @ + remote hostname + \0 */
```

Comment:
I think too many lines inside case, what about simpify the case itself by making function, like csql_set_plan_dump ()
For example,

```
case S_CMD_CONNECT:
  error_code = csql_connect ((argument[0] == '\0') ? NULL : argument);
  if (error_code != NO_ERROR)
    {
       return error_code;
    }

```... near the end ...
```
int
csql_connect (const char *arg_str)
{
 ...
}
```

Reply:
Thank you. I will apply it.